htlcswitch/link.go

```diff
@@ -3029,9 +3029,11 @@ func (l *channelLink) processRemoteAdds(fwdPkg *channeldb.FwdPkg,
 		}
 
 		// Instantiate an error encrypter based on the extracted
-		// encryption parameters.
+		// encryption parameters. Don't assume attributable errors,
+		// because first the resolution format needs to be decoded from
+		// the onion payload.
```

Comment:
Fallback to legacy format is probably fine. If these happens to be an attributable error route, the upstream node will substitute with an attributable error?

Reply:
This should be added to the spec.